src/Presenters/FaqPresenters.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Ourgarage\Faq\Presenters;
+
+use Illuminate\Database\QueryException;
+use Ourgarage\Faq\Models\Category;
+
+class FaqPresenters
+{
+    /**
+     * Get all categories of FAQ
+     *
+     * @return mixed
+     */
+    public function getAllCategories()
+    {
+        return Category::paginate(20);
+    }
+
+    /**
+     * Create new category of FAQ
+     *
+     * @param $data
+     * @return bool
+     */
+    public function createCategory($data)
+    {
+        try {
```

Comment:
This looks very weird :)
Also you just returning `true` in case of error so no one will be able to handle this error. I would catch such errors in ExceptionHandler.